ui/src/components/Controls/index.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/styles'
+import EditControl from './Edit'
+import DownloadControl from './Download'
+
+const useStyles = makeStyles({
+  controls: {
+    flexGrow: 0,
+    minWidth: 200,
+  },
+})
+
+const Controls = ({
+  windowOptions,
+  window,
+  setWindow,
+  aggregationOptions,
+  aggregateBy,
+  setAggregateBy,
+  accumulateOptions,
+  accumulate,
+  setAccumulate,
+  idleOptions,
+  idle,
+  setIdle,
+  title,
+  cumulativeData,
+  currency,
+  titleField,
+  filters,
+  setFilters,
+  clearContext,
+  context,
+}) => {
+  const classes = useStyles()
+
+  let reportAggregateBy = aggregateBy
+  if (context.length > 0) {
+    reportAggregateBy = context[0].property.toLowerCase()
+  }
+
+  return (
+    <div className={classes.controls}>
+      <EditControl
+        windowOptions={windowOptions}
+        window={window}
+        setWindow={setWindow}
+        aggregationOptions={aggregationOptions}
+        aggregateBy={aggregateBy}
+        setAggregateBy={setAggregateBy}
+        accumulateOptions={accumulateOptions}
+        accumulate={accumulate}
+        setAccumulate={setAccumulate}
+        idleOptions={idleOptions}
+        idle={idle}
+        setIdle={setIdle}
+        filters={filters}
+        setFilters={setFilters}
+        currency={currency}
+        clearContext={clearContext}
+      />
```

Comment:
I think that'd be pretty straightforward and agree that is nicer from a UI/UX perspective.

I had thought maube it makes sense to leave the UX a little more painful here for the sake of consistency. Like, if someone goes from OSUI to trying out the closed-source product, that menu is in the same place but with more stuff.

You could also make a case though that the best way to *show* them that there are new options in the closed source is to make that experience different (OS, all options are laid out flat, CS "where are my options? oh they're in this buTTON WITH NEW OPTIONS?!").

So now that you say it, I'm on board. Happy to make the change

Reply:
Haha I like thinking this way about what users' experiences might be. Consistency is generally a good argument, too, but in this case I think we are okay with a break because the closed-source UI will always have more features, and in some cases the OS might even have features the CS doesn't have (difficult to think of one, but possible). If we were talking "free" to "enterprise", where fluidity between tiers was expected, then consistency feels way more important to me.

At any rate, thank you for the changes--I think it's looking absolutely fantastic.